packages/ui/src/Table/common/Table.tsx

```diff
@@ -300,16 +313,47 @@ const DataTable = <TData extends { id: string | number }>({
                 data-state={row.getIsSelected() && "selected"}
                 data-id={row.original.id ?? row.id}
               >
-                {row.getVisibleCells().map((cell) => (
-                  <TableCell
-                    key={cell.id}
-                    data-label={cell.column.id}
-                    data-align={cell.column.columnDef.align || "left"}
-                    className={cell.column.id ? `column-${cell.column.id}` : ``}
-                  >
-                    {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                  </TableCell>
-                ))}
+                {row.getVisibleCells().map((cell) => {
+                  if (cell.column.columnDef.tooltip) {
+                    return (
+                      <TableCell
+                        key={cell.id}
+                        data-label={cell.column.id}
+                        data-align={cell.column.columnDef.align || "left"}
+                        className={
+                          cell.column.id ? `column-${cell.column.id}` : ``
+                        }
```

Comment:
This part looks same as below when tooltip is not enabled. Should we use the condition to check and render only the content inside?